### PR TITLE
[DX-323] Update MongoDB version support for v5.0.2

### PR DESCRIPTION
**Pull request by carolwkc**
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/planning-for-production/database-settings/mongodb.md [hunk 1]
```diff
@@ -11,7 +11,19 @@ weight: 2
...
+### Choose a MongoDB driver
+
+From Tyk 5.0.2, we added an option to use the official MongoDB Go driver to connect to MongoDB. You can configure which driver to use with the MongoDB driver option:
+* [Configure Dashboard MongoDB driver]({{< ref "/tyk-dashboard/configuration" >}})
+* [Configure MDCB MongoDB driver]({{< ref "/tyk-multi-data-centre/mdcb-configuration-options" >}})
+* [Configure Pump MongoDB driver](https://github.com/TykTechnologies/tyk-pump#driver-type)
+
+With mongo-go driver, we support latest version of MongoDB (5.0.x and v6.0.x) and also features such as "+srv" connection string and SCRAM-SHA-256. For more details, visit the MongoDB doc:
+* [Connection Guide](https://www.mongodb.com/docs/drivers/go/v1.11/fundamentals/connection/)
+* [Authentication Mechanisms](https://www.mongodb.com/docs/drivers/go/v1.11/fundamentals/auth/)
```
Comment (sredxny) [inline tyk-docs/content/planning-for-production/database-settings/mongodb.md]: maybe worth to mention that if you're using mongo versions lower than 3.6 then you should use mgo? am looking to provide information like: when should I use mgo vs mongo-go

Comment (sredxny) [reply tyk-docs/content/planning-for-production/database-settings/mongodb.md]: well, I see now that this information is in: `{{< mongodb-versions >}}`
`Maybe should we say something like: to know which driver fits to your need check {{< mongodb-versions >}}` ?

Comment (carolwkc) [reply tyk-docs/content/planning-for-production/database-settings/mongodb.md]: Thanks @sredxny . Updated inline

**Review by sredxny — APPROVED** [review]

File: tyk-docs/content/planning-for-production/database-settings/mongodb.md [hunk 2]
```diff
@@ -11,7 +11,21 @@ weight: 2
...
+{{< mongodb-versions >}}
+
+### Choose a MongoDB driver
+
+From Tyk 5.0.2, we added an option to use the official MongoDB Go driver to connect to MongoDB. You can configure which driver to use with the MongoDB driver option:
+* [Configure Dashboard MongoDB driver]({{< ref "/tyk-dashboard/configuration" >}})
+* [Configure MDCB MongoDB driver]({{< ref "/tyk-multi-data-centre/mdcb-configuration-options" >}})
+* [Configure Pump MongoDB driver](https://github.com/TykTechnologies/tyk-pump#driver-type)
+
+We recommend using mongo-go driver if you are using MongoDB 4.4.x+. For MongoDB versions prior to 4.4, please use mgo driver.
```
Comment (rewsmith) [inline tyk-docs/content/planning-for-production/database-settings/mongodb.md]:
```suggestion
We recommend using the mongo-go driver if you are using MongoDB 4.4.x+. For MongoDB versions prior to 4.4, please use the mgo driver.
```

File: tyk-docs/content/planning-for-production/database-settings/mongodb.md [hunk 3]
```diff
@@ -11,7 +11,21 @@ weight: 2
...
+
+With mongo-go driver, we support latest version of MongoDB (5.0.x and v6.0.x) and also features such as "+srv" connection string and SCRAM-SHA-256. For more details, visit the MongoDB doc:
```
Comment (rewsmith) [inline tyk-docs/content/planning-for-production/database-settings/mongodb.md]:
```suggestion
With the mongo-go driver, we support the latest versions of MongoDB (5.0.x and v6.0.x) and also features such as the "+srv" connection string and SCRAM-SHA-256. For more details, visit the MongoDB doc:
```

File: tyk-docs/content/tyk-dashboard/database-options.md [hunk 4]
```diff
@@ -12,32 +12,26 @@ menu:
 ## Introduction
 Tyk Dashboard reuqires a persistent datastore for its operations. By default MongoDB is used. From Tyk v4.0, we also support PostgreSQL.
```
Comment (rewsmith) [inline tyk-docs/content/tyk-dashboard/database-options.md]:
```suggestion
Tyk Dashboard requires a persistent datastore for its operations. By default MongoDB is used. From Tyk v4.0, we also support PostgreSQL. 
```

File: tyk-docs/content/tyk-dashboard/database-options.md [hunk 5]
```diff
@@ -12,32 +12,26 @@ menu:
 ## Introduction
 Tyk Dashboard reuqires a persistent datastore for its operations. By default MongoDB is used. From Tyk v4.0, we also support PostgreSQL. 
 
-### MongoDB Support Versions and Drop-in Replacement
-MongoDB is our default storage option. We support the following versions:
-* [MongoDB](https://www.mongodb.com) 4.4.x
+## MongoDB Support Versions and Drop-in Replacement
```
Comment (rewsmith) [inline tyk-docs/content/tyk-dashboard/database-options.md]:
```suggestion
## MongoDB Supported Versions and Drop-in Replacement
```

File: tyk-docs/content/tyk-dashboard/database-options.md [hunk 6]
```diff
@@ -12,32 +12,26 @@ menu:
...
-* [Azure CosmosDB for MongoDB](https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/introduction) 3.6 and 4 engine
+### Configuring MongoDB
 
-Please check [here]({{< ref "planning-for-production/database-settings/mongodb.md" >}}) for production configurations.
+Please check [here]({{< ref "planning-for-production/database-settings/mongodb.md" >}}) for MongoDB driver and production configurations.
 
-### PostgreSQL Support Versions and Drop-in Replacement
-From Tyk 4.0, you can use PostgreSQL as your datastore. We support the following versions:
-* [PostgreSQL](https://www.postgresql.org) version 11.x, 12.x, 13.x, 14.x, 15.x
+## PostgreSQL Support Versions and Drop-in Replacement
```
Comment (rewsmith) [inline tyk-docs/content/tyk-dashboard/database-options.md]:
```suggestion
## PostgreSQL Supported Versions and Drop-in Replacement
```

File: tyk-docs/content/tyk-dashboard/database-options.md
Comment (rewsmith) [inline tyk-docs/content/tyk-dashboard/database-options.md:1]: The note under MongoDB Support(ed) Versions isn't formatted correctly

**Review by rewsmith — DISMISSED** [review]

**Comment by dcs3spp** [comment]
@caroltyk I have added a markdown file to _content/shared_ folder. Any files in this folder can be referenced from other markdown documents. The file is called mongodb-versions-include.md. This way the markdown is automatically rendered to HTML.

```
---
title: MongoDB Support
menu:
  main:
    parent: "Tyk Dashboard"
---

The following Mongo versions are used

{{< include "mongodb-versions-include" >}}
```

File: tyk-docs/content/tyk-dashboard/database-options.md
Comment (carolwkc) [reply tyk-docs/content/tyk-dashboard/database-options.md:1]: The formatting issue is resolved with @dcs3spp 's help :)

**Review by dcs3spp — APPROVED** [review]
lgtm Carol

**Comment by dcs3spp** [comment]
@rewsmith can you please check if review changes requested are ok now. The PR will automatically merge once approved.

**Review by sredxny — APPROVED** [review]
lgtm 🚀

**Pull request merged**